src/platform/lumin-runtime/mxs-landscape-app.js

```diff
@@ -24,7 +25,7 @@ export class MxsLandscapeApp extends LandscapeApp {
   onAppStart(arg) {
     const container = {
       controller: {
-        getRoot: () => ({ addChild: (child) => console.log('App container - adding child (scene)') })
+        getRoot: () => ({ addChild: (child) => logInfo('App container - adding child (scene)') })
```

Comment:
do we need this log?  This log is only useful for us as developers of the framework.

Reply:
The severity is `Info`. It will not be printed to the log unless the framework is configured to print all types of logs. By default the severity is `Exception` only.